README.md

```diff
@@ -103,13 +118,38 @@ Sample setting:
 "checker-framework.languageserver_repo": "checker-framework-languageserver"
 ```
 
+## Requirements 
+
+Node.js and yarn are required.
+
+## Developer's Guide
+
+During development, you can simply start the program from the command line and run as a new VS Code instance:
+
+1.  Install dependencies using `yarn install`.
+2.  Open up this project (`checker-framework-vscode`) in VS Code and start a new instance by clicking on `Run`
+    in the menu bar, then `Start Debugging`. 
+4.  Open a `.java` file (this can be a simple Hello World example) in the `[Extension Development Host]` instance of VSCode.
+
+The manual method involves the following steps:
```

Comment:
This sentence seems a bit lost. In what sense is this a "manual" method?

Reply:
I believe this is referring to the initial set-up. I will reword it for clarity